src/app/components/EditorInterface.tsx

```diff
@@ -243,7 +259,7 @@ const EditorInterface = ({ settings, token }: IEditorInterface) => {
       (appState?.appState.settings as ISettings).backgroundColor,
     )
     appState?.setBannerImgUrl(
-      (appState?.appState.settings as ISettings).bannerImage.url,
+      (appState?.appState.settings as ISettings).bannerImage?.url || '',
```

Comment:
Lets avoid typecasting when possible, we can enforce the type with Zod. Can be done later

Reply:
left this for later